lib/main.coffee

```diff
@@ -108,6 +108,7 @@ module.exports =
         parameters = @parameters.filter (item) -> item
         command = @command
         confFile = helpers.find(path.dirname(filePath), ['phpcs.xml', 'phpcs.xml.dist', 'phpcs.ruleset.xml'])
+        cwd = path.dirname(confFile) if confFile
```

Comment:
It will, yes. But setting the cwd to undefined seems to behave the same way as not setting it at all, and I thought that was better than wrapping the exec call in an if statement and only add the cwd parameter if it's not undefined. I'm fine with changing the default to the directory of the file instead if you think that's better.